# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/storage-map/Cargo.toml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/storage-map/Cargo.toml b/third_party/rust/storage-map/Cargo.toml
deleted file mode 100644
--- a/third_party/rust/storage-map/Cargo.toml
+++ /dev/null
@@ -1,27 +0,0 @@
-# THIS FILE IS AUTOMATICALLY GENERATED BY CARGO
-#
-# When uploading crates to the registry Cargo will automatically
-# "normalize" Cargo.toml files for maximal compatibility
-# with all versions of Cargo and also rewrite `path` dependencies
-# to registry (e.g., crates.io) dependencies
-#
-# If you believe there's an error in this file please file an
-# issue against the rust-lang/cargo repository. If you're
-# editing this file be aware that the upstream Cargo.toml
-# will likely look very different (and much more reasonable)
-
-[package]
-edition = "2018"
-name = "storage-map"
-version = "0.3.0"
-authors = ["Dzmitry Malyshau <kvarkus@gmail.com>"]
-description = "Concurrent append-only map storage"
-homepage = "https://github.com/kvark/storage-map"
-documentation = "https://docs.rs/crate/storage-map"
-keywords = ["hashmap", "map", "concurrent", "lock"]
-license = "Apache-2.0"
-repository = "https://github.com/kvark/storage-map"
-[dependencies.lock_api]
-version = "0.4"
-[dev-dependencies.parking_lot]
-version = "0.11"